gettsim/benefits/elterngeld.py

```diff
@@ -0,0 +1,215 @@
+        percentage = params["elgfaktor"] + number_steps * params["elg_perc_correct"]
+
+    elif considered_wage > params["elg_st_2"]:
+        wag_diff = considered_wage - params["elg_st_2"]
+
+        number_steps = wag_diff / params["elg_perc_correct_step"]
+        percentage = max(
+            params["elgfaktor"] - number_steps * params["elg_perc_correct"],
+            params["perc_deduct_limit"],
+        )
+
+    else:
+        percentage = params["elgfaktor"]
+
+    return percentage
+
+
+def calc_net_wage(person):
+    """ Calculating the net wage of any person given taxes and social security
+    contributions.
+
+    """
+    net_wage = (
+        person["m_wage"] - person["incometax"] - person["soli"] - person["svbeit"]
+    )
+
+    return net_wage
+
+
+def calc_geschw_bonus(elterngeld_calc, params):
+    """ Calculating the bonus for siblings.
+
+
+    According to § 2a parents of siblings get a bonus.
+    """
+    bonus_calc = params["elg_geschw_bonus_share"] * elterngeld_calc
+    bonus = max(
+        min(bonus_calc, params["elg_geschw_bonus_max"]), params["elg_geschw_bonus_min"]
+    )
+    return bonus
+
+
+def check_eligibilities(household, params):
+    """Check if a parent is eligible for elterngeld. If so, then also check if it is
+    eligible for multiple or sibling bonus.
+
+    """
+    household["elternzeit_anspruch"] = False
+    household["geschw_bonus"] = False
+    household["num_mehrlinge"] = 0
+
+    children = household[household["child"]]
+    # Are there any children
+    if len(children) > 0:
+        youngest_child = children[
+            (children["byear"] == np.max(children["byear"]))
+            & (children["bmonth"] == np.max(children["bmonth"]))
+        ]
+        # Get the birthdate of the youngest child(If "Mehrlinge", then just take one)
+        birth_date = datetime.date(
+            day=youngest_child["bday"].iloc[0].astype(int),
+            month=youngest_child["bmonth"].iloc[0].astype(int),
+            year=youngest_child["byear"].iloc[0].astype(int),
+        )
+        age_youngest_child = relativedelta.relativedelta(params["date"], birth_date)
+        # The child has to be below the 14th month
+        eligible_age = (
+            age_youngest_child.years * 12 + age_youngest_child.months
+        ) <= params["max_joint_months"]
+        # The parents can only claim up to 14 month elterngeld
+        eligible_consumed = (
+            youngest_child["elterngeld_mon_mut"].iloc[0]
+            + youngest_child["elterngeld_mon_vat"].iloc[0]
+        ) <= 14
+
+        if eligible_age & eligible_consumed:
+            # Each parent can't claim more than 12 month
+            eligible = ~household["child"] & (
+                household["elterngeld_mon"] <= params["max_months"]
+            )
+
+            household.loc[eligible, "elternzeit_anspruch"] = True
+
+            if (len(children[(params["year"] - children["byear"]) < 3]) == 2) | (
```

Comment:
please add one comment on what this if-condition contains

Reply:
Added comment